flow/record/fieldtypes/__init__.py

```diff
@@ -50,9 +56,13 @@ def flow_record_tz(*, default_tz: str = "UTC") -> Optional[ZoneInfo | UTC]:
     Returns:
         None if ``FLOW_RECORD_TZ=NONE`` otherwise ``ZoneInfo(FLOW_RECORD_TZ)`` or ``UTC`` if ZoneInfo is not found.
     """
+    if not HAS_ZONE_INFO:
+        return UTC
+
     tz = os.environ.get("FLOW_RECORD_TZ", default_tz)
     if tz.upper() == "NONE":
         return None
+
```

Comment:
Can you insert something like this here:

```python
if not HAS_ZONE_INFO and tz != "UTC":
    warning.warn("Cannot use FLOW_RECORD_TZ due to missing zoneinfo module, defaulting to 'UTC'.")
    return UTC
```

Reply:
Good catch, then something like this?

```python
if not HAS_ZONE_INFO:
    if tz != "UTC":
        warning.warn("Cannot use FLOW_RECORD_TZ due to missing zoneinfo module, defaulting to 'UTC'.")
    return UTC
```

So always return UTC when zoneinfo is not available, and only print the warning if the user actually configured FLOW_RECORD_TZ to something non UTC.